dspace-api/src/main/java/org/dspace/authenticate/ShibAuthentication.java

```diff
@@ -33,6 +33,7 @@
 import org.dspace.content.factory.ContentServiceFactory;
 import org.dspace.content.service.MetadataFieldService;
 import org.dspace.content.service.MetadataSchemaService;
+import org.dspace.core.ConfigurationManager;
```

Comment:
Just a note here. You shouldn't need to import the ConfigurationManager, as this class already has access to the `ConfigurationService`.  The ConfigurationManager is deprecated in favor of the ConfigurationService.  So, you should remove this import and just use `configurationService` instead.